win32/deluge-bbfreeze.py

```diff
@@ -3,12 +3,11 @@
 import shutil
 import sys
 
-import gtk
-
-import bbfreeze.recipes
+import bbfreeze
 import deluge.common
 import icon
-from bbfreeze import Freezer
+from bbfreeze import Freezer, recipes  # NOQA
+from gi.repository import Gtk  # NOQA
```

Comment:
Please do not use `#NOQA` it is failing because you have made a mistake. `flake8` is trying to help you write good code and `isort` puts the import lines in the correct place.